data-r2dbc/build.gradle

```diff
@@ -0,0 +1,97 @@
+configurations.all {
```

Comment:
This will change the group id for this artefact from `io.micronaut.r2dbc` to `io.micronaut.data` 

If this is how it is going to be we will have to make sure Micronaut Launch is updated to refer to the new group ID and provide update instructions to users.

Reply:
Yes, I expect it to be: `io.micronaut.data:micronaut-data-r2dbc` in the same format as jdbc